meson.build

```diff
@@ -478,6 +478,10 @@ features = {
   'ml-agent-support': {
     'extra_deps': [ ml_agent_dep ],
     'project_args': { 'ENABLE_ML_AGENT' : 1 }
+  },
+  'onnxruntime-support': {
+    'target': 'libonnxruntime',
```

Comment:
Is the pkgconfig name really ```libonnxruntime```? Have you tested with ```-Donnxruntime-support=enabled```?

Reply:
Yes, the pkgconfig name is libonnxruntime.so